packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -477,10 +479,24 @@ export class ConnectionFlow {
     await this.launch()
   }
 
+  /**
+   * Add clientSide attribute to the konnector manifest according to the context
+   *
+   * @todo When the stack handles the clientSide attribute, this won't be needed anymore
+   */
+  withClientSide(konnector) {
+    const clientSideSlugs = get(window, 'cozy.clientSideSlugs', [])
+    const clientSide = clientSideSlugs.includes(konnector.slug)
+
+    return { ...konnector, clientSide }
+  }
+
   /**
    * Launches the job and sets everything up to follow execution.
    */
   async launch({ autoSuccessTimer = true } = {}) {
+    const konnector = this.withClientSide(this.konnector)
```

Comment:
I don't think we need this code anymore this the trigger has the `client` attribute, right? (again relying on this.konnector will break stuff see previous comment). 

Generally we should not add a dep to the konnector when launching a trigger

Reply:
Same as https://github.com/cozy/cozy-libs/pull/1296#discussion_r689681060 . Harvest does not know it should create a client trigger if it cannot find the clientSide attribute in io.cozy.konnectors.

Reply:
Done. This is now possible since https://github.com/cozy/cozy-stack/pull/3113